woocommerce/google-listings-and-ads
# Update Plugin Icons

### Changes proposed in this Pull Request:

Closes #1644  

Replace old google Shopping icons with the new ones with the G letter logo

### Screenshots

<img width="689" alt="Screenshot 2022-08-25 at 18 27 59" src="https://user-images.githubusercontent.com/5908855/186692398-566c728e-9376-42df-9f24-dd08883cf6bd.png">

<img width="321" alt="Screenshot 2022-08-25 at 18 28 04" src="https://user-images.githubusercontent.com/5908855/186692419-0a446d67-0368-4c0c-8bca-12c11204ed85.png">


### Detailed test instructions:

- Check the new images in `wordpress_org_assets`
- Check the image icon.svg - 128x128 and with white background
- Check the image icon-no-bg.svg -  128x128 and with NO background
- Check the image icon128x128.png - 128x128 and with white background
- Check the image icon256x256.png - 256x256 and with white background
- Check that there is no icons in the code referring to the old icons 

### Changelog entry

>  Tweak - Updated plugin icons
